website/content/en/docs/building-operators/ansible/quickstart.md

```diff
@@ -1,96 +1,92 @@
 ---
-title: Ansible Operator QuickStart
-linkTitle: QuickStart
-weight: 2
+title: Quickstart for Ansible-based Operators
+linkTitle: Quickstart
+weight: 100
+description: A simple set of instructions that demonstrates the basics of setting up and running a Ansible-based operator.
 ---
-## Prerequisites
-
-- [docker][docker_tool] version 17.03+.
-- [kubectl][kubectl_tool] version v1.11.3+.
-- [Ansible Operator SDK Installation][ansible-operator-install] v1.0.0+
-- Access to a Kubernetes v1.11.3+ cluster.
 
-## Creating a Project
+This guide walks through an example of building a simple memcached-operator powered by [Ansible][ansible-link] using tools and libraries provided by the Operator SDK.
 
-Create a directory, and then run the init command inside of it to generate a new project.
- 
-```sh
-$ mkdir $GOPATH/src/memcached-operator
-$ cd $GOPATH/src/memcached-operator
-$ operator-sdk init --plugins=ansible
-```
+## Prerequisites
 
-## Creating an API
+- [Install `operator-sdk`][operator_install] and the [Ansible prequisites][ansible-operator-install] 
+- Access to a Kubernetes v1.16.0+ cluster.
+- User logged with admin permission. See [how to grant yourself cluster-admin privileges or be logged in as admin][role-based-access-control].
 
-Let's create a new API with a default role for it:
+## Quickstart Steps
 
-```sh
-$ operator-sdk create api --group cache --version v1 --kind Memcached --generate-role 
-```
+### Create a project
 
-## Applying the CRDs into the cluster:
-
-To apply the `Memcached` Kind(CRD): 
+Create and change into a directory for your project. Then call `operator-sdk init`
+with the Ansible plugin to initialize the [base project layout][layout-doc]:
 
 ```sh
-$ make install
+mkdir memcached-operator
+cd memcached-operator
+operator-sdk init --plugins=ansible
```

Comment:
```suggestion
operator-sdk init --plugins=ansible --domain=example.com
```
I've been bitten by allowing the domain to default to my.domain, and had to start over.